examples/gsInterpolateMap.cpp

```diff
@@ -113,10 +113,10 @@ int main(int argc, char* argv[])
         tBasis = gsBSplineBasis<>::make(KV);
         break;
     case 2:
-        tBasis = safe( new gsTensorBSplineBasis<2>(KV,KV) );
+        tBasis = memory::unique_ptr<gsTensorBSplineBasis<2> >(new gsTensorBSplineBasis<2>(KV,KV));
```

Comment:
I don't like that. We have wrapper functions not to be required to specify the template parameter. That's even what std does.

We might change this to memory::make_unique(...) if desired.